frontend/src/pages/device/dialogs/ModeChoiceDialog.vue

```diff
@@ -5,7 +5,7 @@
                 <template v-if="unsupportedVersion">
                     <p>
                         Developer Mode requires Device Agent v0.8.0 or later.
-                        Please update your Device Agent to the latest version.
+                        Please ensure this device has checked in at least once and that it is running the latest version of Device Agent.
```

Comment:
I understand the intent, my concern is the user doesn't know how to make the device check in. Does the following help?

```suggestion
                        Please ensure the Device Agent is up to date and connected to the platform.
```

Reply:
better :)